cmd/gotypeconv/main.go

```diff
@@ -21,6 +21,12 @@ import (
 	"golang.org/x/tools/go/loader"
 )
 
+var unused = 1
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var unused is unused (U1000)